src/setup/devnet/index.js

```diff
@@ -404,35 +404,80 @@ export class Devnet {
                         `${ganacheUser}@${ganacheURL.hostname}:~/data`
                     ], {stdio: getRemoteStdio()})
 
-                    // Run ganache in tmux
-                    await execa('ssh', [
+                    // Generate service files
+                    for (let i = 0; i < this.totalNodes; i++) {
+                        await execa('scp', [
+                            `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                            `-i`, `~/cert.pem`,
+                            `${this.config.targetDirectory}/service.sh`,
+                            `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}:~/`
+                        ], {stdio: getRemoteStdio()})
```

Comment:
can we give the copied file a meaningful name?

Reply:
Not sure what you mean. It's just a bash script that generates service files that are executed as systemd processes.